src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -631,7 +666,11 @@ def _update_instance(instance, part, path):
                 show_options(instance, part, path)
 
             if '=' in index.group(1):
-                key, value = index.group(1).split('=')
+                key, value = index.group(1).split('=', 1)
```

Comment:
Any reason we don't use the better `_split_key_value_pair`? Even that might be overkill, but for consistent and maintenance, I suggest we use the better one, or put a comment here for reasons why we don't need it